Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,501 @@
+open to all majors because most of these skills can be taught in at least a basic manner.
+
+The basic nozzle theory required here can be adequately learned by reading the introductory chapters of \textit{Rocket Propulsion Elements} by Sutton\cite{RPE}. Chapter 1 through 4 will
+provide a sufficient basis of understanding for this project.
+
+ The CAD software of choice is Solidworks. It is very common for industry and has all the features that the team
+will need throughout the process. The education edition also includes the simulation premium package that can allow the team to run flow simulations if needed. The procedure for
+doing this is complicated but can be learned to a basic degree on YouTube.
+
+The basic electrical engineering including setting up and arduino or raspberry pi to read out the data from the load cell and turning the solenoid on and off. It
+should not be difficult for a first year or second year EE or EET to set this up. If it proves to be difficult, the team can each out to older students or professors for assistance.
+Dr.\ Patru would certainly be a great resource for this kind of help.
+
+The basic coding abilites include writing a MATLAB script to analyze and plot data. MATLAB is easy to learn and is very powerful. It is a very common choice for data analysis for mechancial engineering
+projects where the core skill set is not computer science. This is not hard for anyone who has coded before.
+
+The machining will be the most difficult to learn if there are no MEs or METs that have taken the proper courses. Machining can be extremely dangerous and cannot be done
+without the proper safety training. If no one is trained on the machines one can ask a machinist to make simple parts or submit a job at the Earl Brickman Machine Tools and Manufacturing Lab.
+They have professional engineers there that can make nearly anything one could need. But their timeline slips often, so leave an appropriate amount of time for them to make the part
+so it does not interfere with the projects timeline.
+
+\subsection{Funding Requirements}
+  % Estimate costs that would be needed to meet objectives.
+Fortunately, most of the materials for the test stand have been purchased. This eliminates much of the system cost. The most costly part of the project
+will be the propulsion system (the regulator, solenoid, nozzle, piping, and interfaces). A preliminary budget that includes margin for cost overruns is \$600.
+The total cost was approximately \$450. That would be the last part of the system needed before testing can occur. Accounting for unforseen cost then rounding upwards,
+the consquentical estimated cost is put at \$600 for the entirty of the project. A bill of materials is displayed in Appendix B. The table, displayed below, outlines
+the most expensive parts that the team will need to purchase for the propulsion system, these parts are worst-case scenario in cost and measures will be taken to
```

Comment:
This section can be shortened, in that estimated Bill of Materials cost is $450 but $600 is requested to cover unexpected costs.